scipy/sparse/linalg/eigen/arpack/tests/test_arpack.py

```diff
@@ -774,9 +774,12 @@ def reorder(args):
         A = np.random.RandomState(52).randn(n, m)
         L = CheckingLinearOperator(A)
 
-        v0 = np.ones(min(A.shape))
+        for solver in [None, 'arpack', 'lobpcg', 'propack']:
+            if solver == 'propack':
+                v0 = np.ones(n)
+            else:
+                v0 = np.ones(min(A.shape))
```

Comment:
`propack` must work differently from the other solvers as it always needs `v0.size == A.shape[0]`